trunner/runners/STM32L4Runner.py

```diff
@@ -0,0 +1,95 @@
+#
+# Phoenix-RTOS test runner
+#
+# STM32l4 runner
+#
+# Copyright 2021 Phoenix Systems
+# Authors: Mateusz Niewiadomski
+#
+
+import os
+import subprocess
+import sys
+import time
+
+import pexpect.fdpexpect
+import serial
+from pexpect import TIMEOUT
+
+from .common import DeviceRunner, Color
+from .common import _BOOT_DIR
+
+
+class STM32L4Runner(DeviceRunner):
+    """ This class provides interface to run test case on STM32L4 target using RaspberryPi.
```

Comment:
Note for the future: when writing docstring it's not needed to start with "This class" as I have the class name just above :).

Also, my personal preference is to format it like:

```python
def fun():
    """ single line docstring """

def fun3():
    """ Single-line summary
    
    Multiline detailed description
    with possible examples
    and description of input parameters.
    """

def fun3():
    """
    Multiline docstring.
    With many newlines, and possible indentations, eg. use examples:
        if fun3():
            return fun()
    """
```
(no extra indentation on multiline docstrings)

PEP-257 says these variants are equal so it boils down to the possible increased readability only.

See sample docstrings in bigger python projects, eg.: https://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html#docstrings

Reply:
Thank you for providing example :) 

Docstrings changed to conform with convention